winter/web/pagination/order_by.py

```diff
@@ -3,7 +3,7 @@
 from .check_sort import check_sort
 from .order_by_annotation import OrderByAnnotation
 from .parse_sort import parse_sort
-from ...core.annotation_decorator import annotate_method
+from winter.core.annotation_decorator import annotate_method
```

Comment:
I like when absolute imports is a separate group of imports, maybe joined to a group of 3rd party libraries:

```
import enum

from django.conf import settings
from winter.core.annotation_decorator import annotate_method

from .check_sort import check_sort
from .order_by_annotation import OrderByAnnotation
from .parse_sort import parse_sort
```

Reply:
It's formatted so by IDE because it's from the same library.